### ISSUE TITLE
Extend checks for parachutes

### ISSUE THREAD

**Issue by WDBarnes** [jkoritzinsky/Extensive-Engineer-Report#4]
Check for RealChuteModule (from Real Chutes) and RealChuteFAR (from FAR) in addition to the stock ModuleParachute

**Comment by jkoritzinsky** [comment]
Are you referring to the stock checks for parachutes? I can't modify the stock checks or even remove them from the report (the game code doesn't make it possible). If I could, I would.

**Comment by WDBarnes** [comment]
Well, yes and no. When using FAR or Real Chutes, the engineer report issues warnings about missing parachutes even when parachutes are attached because both of those mods use their own custom modules for parachutes. I was hoping there might be a way to check for those custom modules but even if you did, it looks like the game will still check for (and not find) the stock chutes.

**Comment by jkoritzinsky** [comment]
Yep. That's what will happen. I'll keep this open just in case they open it up or if I make tests that involve parachutes.
